dotCMS/src/main/webapp/html/portlet/ext/contentlet/edit_contentlet.jsp

```diff
@@ -85,6 +85,10 @@
 	<script src="/dotcms-block-editor/main.js" type="module"></script>
 <!--   End dotCMS Block Editor -->
 
+<!--  dotCMS Custom Field Bridge -->
+<script src="/edit-content-bridge/edit-content-bridge.js"></script>
```

Comment:
Should this be under /html/js?  or will it be used outside of the dotcms ui?

Reply:
Being JSP, it takes the same place where we put the other libraries that are built and injected into the Edit Content DOJO. Which is in `/core/dotCMS/src/main/webapp` but if the correct path is `/core/dotCMS/src/main/webapp/html/js` I can inject there

Reply:
Change to put the file into` /core/dotCMS/src/main/webapp/html/js/legacy_custom_field `

Reply:
![CleanShot 2025-01-24 at 13 52 42@2x](https://github.com/user-attachments/assets/b9080986-912d-45bf-8909-518b5704a129)